src/Tribute.js

```diff
@@ -256,7 +264,9 @@ class Tribute {
                 li.addEventListener('mouseenter', (e) => {
                   let li = e.target;
                   let index = li.getAttribute('data-index')
-                  this.events.setActiveLi(index)
+                  if (e.movementX !== 0 && e.movementY !== 0) {
```

Comment:
Fixes bug that caused setActiveLi to be called when mouse pointer inside one of the items and scrolling though keyboard, caused improper highlighting.

![ezgif com-video-to-gif](https://user-images.githubusercontent.com/21201812/52480095-c03e2180-2bd0-11e9-9c54-e67a6f465728.gif)